prep_local

```diff
@@ -10,7 +10,7 @@ set -o errexit -o nounset -o pipefail
 : ${VIRTUAL_ENV?"Must be run in a python virtual environment"}
 
 # Install the latest version of pip
-pip install pip==8.1.2
+pip install --upgrade pip
```

Comment:
I will revert it in a separate PR. Let's not confuse it with this one.  thank you.